HW1/Task1.py

```diff
@@ -0,0 +1,36 @@
+def Checkint(name):
+    temp = 0
+    while temp != 1:
+        n = input(f"введите число {name} - ")
+        try:
+            count = int(n)
+            if count < 0 or count > 100000:
+                print("Введите число от 0 до 100 тысяч")
+            else:
+                temp = 1
+        except ValueError:
+            print("Введено не число!")
+    return count
+
+
+def task1():
+    # Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+    # Дано a, b, c - стороны предполагаемого треугольника.
+    # Требуется сравнить длину каждого отрезка-стороны с суммой двух других.
+    # Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника с такими сторонами не существует.
+    # Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+    a = Checkint("a")
+    b = Checkint("b")
+    c = Checkint("c")
+    if a + b <= c or a + c <= b or b + c <= a:
+        print("треугольника с такими сторонами не существует !")
+    elif a == b and b == c:
+        print("Треугольник равносторонний")
+    elif a == b and a != c or a == c and a != b or b == c and b != a:
```

Comment:
Исключённые условия не нужны.
```suggestion
    elif a == b or a == c or b == c:
```